plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -192,14 +203,18 @@ public static function output( $post ) {
 					$meta_list = array();
 
 					if ( $payment_method && 'other' !== $payment_method ) {
-						/* translators: %s: payment method */
-						$payment_method_string = sprintf(
-							__( 'Payment via %s', 'woocommerce' ),
-							esc_html( isset( $payment_gateways[ $payment_method ] ) ? $payment_gateways[ $payment_method ]->get_title() : $payment_method )
+						$payment_method_string = wp_kses_post(
```

Comment:
This may not have been possible when this PR was initially written, but it may be better to use this sort of approach, instead of `wp_kses_post()`:

```php
use Automattic\WooCommerce\Internal\Utilities\HtmlSanitizer;
wc_get_container()->get( HtmlSanitizer::class )->sanitize( $payment_method_title );
```

This aligns with what we allow when payment titles are updated (and is a little stricter than `wp_kses_post()`, though still allows for images).

Reply:
Feel free to ignore this request, as we decided to run with `wp_kses_post()` in [this related PR](https://github.com/woocommerce/woocommerce/pull/37481).